schema/va-spec/base/experimental-functional-impact-study-statement-profile-source.yaml

```diff
@@ -0,0 +1,101 @@
+      type:
+        extends: type
+        const: "ExperimentalVariantFunctionalImpactStudyStatement"
+        default: "ExperimentalVariantFunctionalImpactStudyStatement"
+        description: MUST be "ExperimentalVariantFunctionalImpactStudyStatement".
+      subjectVariant:
+        extends: subject
+        oneOf:
+          - $ref: "/ga4gh/schema/vrs/2.x/json/MolecularVariation"
+          - $ref: "/ga4gh/schema/cat-vrs/1.x/json/CategoricalVariant"
+          - $ref: "/ga4gh/schema/gks-core/1.x/json/iriReference"
+        description: A protein or genomic contextual or canonical molecular variant.
+      predicate:  
+        extends: predicate
+        description: >-
+          The relationship this Statement describes between the subject Variant
+          and object Sequence Feature whose function it may alter.
+        const: impactsFunctionOf                
+      objectSequenceFeature:
+        extends: object
+        oneOf:
+          - $ref: "/ga4gh/schema/gks-core/1.x/json/iriReference"
+          - $ref: "/ga4gh/schema/gks-core/1.x/MappableConcept"      # See https://github.com/ga4gh/gks-common/pull/77
+        description: >-
+          The sequence feature (typically a gene or gene product) on whose function the impact 
+          of the subject variant is assessed.
+      studyContextQualifier:
+        description: >-
+          The assay in which the reported variant functional impact was determined - 
+          providing a specific experimental context in which this effect is asserted to hold.
+        $comment: >-
+          The attribute is required in order to explicitly specify an assay or study context in
+          which the reported impact was observed. It's value can be a Document object, or an IRI 
+          pointing to a document that describes the assay/study. This may be a publication, a
+          database record, or a bespoke document.
+        oneOf:
+          - $ref: "/ga4gh/schema/gks-core/1.x/json/Document"
+          - $ref: "/ga4gh/schema/gks-core/1.x/json/iriReference"
+      impactTypeQualifier:           # This qualifier allows more specific types of impacts to be reported - so the profile supports implementations like CIViC, ClinGen, and I think even some MAVE data.
+        description: >-
+          A term describing a specific type of functional impact that the variant is determined
+          to have on the indicated sequence feature (e.g. decreased activity, dominant negative,
+          neomorphic, reduced Ca2+ binding activity).
+        $comment: >-
+          Implementers may use a set of terms to use here that reflect the specific types of
+          functional impact(s) measured by the assay(s) employed. It is recommended that terms
+          are drawn from established terminologies where possible.
+          
+        # TO DO: I feel like it would be good to try and standardize the values used here by defining an enum based on early adopter needs, and growing from there. 
```

Comment:
TODO see comment here.

Reply:
Agree; before adding this property we should identify what is envisioned to fit here.

Reply:
address during 11-13 sync up. we decided it may be premature to lock down the set of terms to be used by both mave and civic for the impactTypeQualifier. If we choose to add this it would need to live in the community profile. We have moved this and commented out how this could be done for future consideration.